content/en/security/agent.md

```diff
@@ -16,6 +16,22 @@ This article is part of a [series on data security][1].
 
 Customers can send data to the Datadog service by using a locally installed [Agent][2] or through our [HTTP API][3]. While use of Datadog does not strictly require use of the Datadog Agent, the vast majority of customers leverage the Agent. This article describes the main security capabilities and features available to customers to ensure their environment is secure.
 
+## Agent distribution
+
+The official repositories and/or binary packages of the Agent are signed. Verify the distribution channel by checking the signature against one of the following public keys:
+
+- APT repo metadata:
+  - https://keyserver.ubuntu.com/pks/lookup?op=hget&search=d1402d39517b9f8888abfc98d6936dab
+  - https://keyserver.ubuntu.com/pks/lookup?op=hget&search=3e8510ce571008616b42bd67916e83f8
+- RPM packages and repo metadata:
+  - https://yum.datadoghq.com/DATADOG_RPM_KEY_20200908.public
+  - https://yum.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public
+  - https://yum.datadoghq.com/DATADOG_RPM_KEY.public (Agent 6 and older only)
```

Comment:
I think we should also show the key fingerprints here, in addition to their location, to give them another means of verification.

Reply:
Is there a "standard" fingerprint representation, though? How do I get it?

Reply:
You can get it with the output of `gpg --show-keys <key_file>`

For instance, 
```
$ gpg --show-keys DATADOG_RPM_KEY_20200908.public
pub   rsa4096 2020-09-08 [SC] [expires: 2024-09-07]
      C6559B690CA882F023BDF3F63F4D1729FD4BF915
uid                      Datadog, Inc. RPM key (2020-09-08) <package+rpmkey@datadoghq.com>
```